asyncio_mqtt/client.py

```diff
@@ -502,15 +528,15 @@ async def publish(  # noqa: PLR0913
 
     @asynccontextmanager
     async def filtered_messages(
-        self, topic_filter: str, *, queue_maxsize: int = 0
+        self, topic_filter: str
```

Comment:
Again, unrelated to this PR. Move into a separate PR. Also, that is a breaking change!